ta/Makefile

```diff
@@ -8,6 +8,8 @@ ifeq ($(out-dir),)
 $(error invalid output directory (O=$(O)))
 endif
 
+-include $(TA_DEV_KIT_DIR)/host_include/conf.mk
```

Comment:
Does it make sense to silently ignore this file if it's missing? The rest of TA devkit is needed in order to be able to compile any way.